optimum/exporters/onnx/model_configs.py

```diff
@@ -1144,6 +1144,187 @@ def outputs(self) -> Dict[str, Dict[int, str]]:
         return common_outputs
 
 
+class DummySpeechT5InputGenerator(DummyInputGenerator):
```

Comment:
It could make sense to move it in `input_generators.py` directly as it's not specific to ONNX https://github.com/huggingface/optimum/blob/8f33e0e751900abf81b9bd427e076e76dff2b35b/optimum/utils/input_generators.py#L106